### PR TITLE
Update formats-data.js

### PR DESCRIPTION
**Pull request by Piexplode**
Updated: Beedrill
Ekans
Sandshrew

Not Updated but should be discussed:
Arbok - to remove all wrap cos it's boring in randbats, replace with Rock Slide or Body Slam?
Pikachu - make tbolt mandatory, remove thunder? Or no?
Raichu - just run the one set [surf/tbolt/agility/hbeam] or other options? [I don't think thunder should be on there though]

### CODE REVIEW

File: mods/gen1/formats-data.js [hunk 1]
```diff
@@ -56,7 +56,8 @@ exports.BattleFormatsData = {
 		tier: "UU"
 	},
 	beedrill: {
-		randomBattleMoves: ["twineedle","hyperbeam","swordsdance","agility"],
+		randomBattleMoves: ["twineedle","hyperbeam","swordsdance","agility","megadrain"],
+		mandatoryMove: "swordsdance"
```
Comment (ascriptmaster) [inline mods/gen1/formats-data.js]: Add a comma. This will fix your build errors, I think.

File: mods/gen1/formats-data.js [hunk 2]
```diff
@@ -56,7 +56,8 @@ exports.BattleFormatsData = {
 		tier: "UU"
 	},
 	beedrill: {
-		randomBattleMoves: ["twineedle","hyperbeam","swordsdance","agility"],
+		randomBattleMoves: ["twineedle","hyperbeam","swordsdance","agility","megadrain"],
```
Comment (xfix) [inline mods/gen1/formats-data.js]: Don't include mandatory move on list of random battle moves.

File: mods/gen1/formats-data.js [hunk 3]
```diff
@@ -96,7 +97,7 @@ exports.BattleFormatsData = {
 		tier: "UU"
 	},
 	ekans: {
-		randomBattleMoves: ["wrap","glare","earthquake","bodyslam","rockslide"],
```
Comment (xfix) [inline mods/gen1/formats-data.js]: Even if boring, Wrap is what defines Ekans, and actually should be mandatory move in my opinion.

**Comment by Zarel** [comment]
Your commit summary also needs to actually summarize what you're committing. :|

**Comment by Piexplode** [comment]
Should I write up in more detail the changes? I've generally spoken with Joim before making changes here.
Also xfix/ascriptmaster's fixes look basically fine

**Comment by ascriptmaster** [comment]
Well, for starters, change your commit title to something that isn't just "Updated this file" to something more meaningful, like "Update Gen 1 Random Battle Movesets" or the like.

**Comment by Joimer** [comment]
Gen 1 Random Battles have been updated recently, also this pull request needs fixing + rebasing but you still need to learn that, so we'll leave that for the next iteration of improvements.

**Pull request closed without merging**